st2debug/setup.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    from ez_setup import use_setuptools
+    use_setuptools()
+    from setuptools import setup, find_packages
+
+setup(
+    name='st2debug',
+    version='0.4.0',
```

Comment:
One option would be to have a version specified inside `VERSION.txt` file inside the repository root.

Then when publishing a package / building stuff we would need to do something like that: `cp VERSION.TXT ; st2<component>/`.

Each setup.py would read version from `VERSION.txt` file which is local to component. For that to work, we would also need to update `MANIFEST.in` to make sure `VERSION.txt` file is included in the distribution.

Keep in mind that this only applies for st2api, st2reactor, st2auth and st2debug components which use the same version. In the future we plan to split st2client into two packages and move it into same repository. After we do that, those two things will probably also be versioned independently so that won't apply there anymore.